### ISSUE TITLE
Multiple instanses

### ISSUE THREAD

**Issue by ortexx** [ghenry22/cordova-plugin-music-controls2#37]
Android 10.

The case:

I build 2 apps with the library. I run one song in the first app and then another one in the second. So there are two notifications with the controls. When i click on any button(at least prev/next) **both** apps get the event message.

I see two solutions:
1) Perfectly, if the library can distinguish the apps some way and send the message only for the necessary one.
2) If it is too problematic, the library can at least send the application name with the event to let the programmer distinguish it in the code. `const application= JSON.parse(action).application;`

**Comment by ghenry22** [comment]
Good catch, I haven't tested that scenario, I will see if we can potentially random generate an APP ID for the notification channel so that it is unique.

The problem at the moment is that a fixed notification channel is used which is always the same.

**Comment by ghenry22** [comment]
I have a fix for this in the development branch at the moment, have implemented a UUID for the channel ID to prevent cross over.

**Comment by ortexx** [comment]
Great news. Waiting for the release(npm)

**Comment by ghenry22** [comment]
weirdly I have tried with randomised channel id and notification id but they still overlap.  I think because both instances are registering broadcast receivers to catch media key events.  I'll try a few more things and see if I can find a way to filter this.

**Comment by ortexx** [comment]
How about my second option in the first message? You can pass the application name from config.xml to the client side. And we can handle messages only for the necessary app.
